misc/tracing-extensions/modules/ballerina-jaeger-extension/src/main/java/org/ballerinalang/observe/trace/extension/jaeger/Constants.java

```diff
@@ -30,17 +32,16 @@ private Constants() {
 
     static final String TRACER_NAME = "jaeger";
 
-    static final String SAMPLER_TYPE_CONFIG = "sampler.type";
-    static final String SAMPLER_PARAM_CONFIG = "sampler.param";
-    static final String REPORTER_LOG_SPANS_CONFIG = "reporter.log.spans";
-    static final String REPORTER_HOST_NAME_CONFIG = "reporter.hostname";
-    static final String REPORTER_PORT_CONFIG = "reporter.port";
-    static final String REPORTER_FLUSH_INTERVAL_MS_CONFIG = "reporter.flush.interval.ms";
-    static final String REPORTER_MAX_BUFFER_SPANS_CONFIG = "reporter.max.buffer.spans";
+    private static final String JAEGER_CONFIG_TABLE = CONFIG_TABLE_TRACING + ".jaeger";
```

Comment:
Are all these Jaeger specific configurations? We discussed that we don't need implementation specific config table headers, right?

Reply:
It was later agreed that jaeger should be added to the configurations. So, this is okay.